packages/lexical-editor-pb/tsconfig.json

```diff
@@ -18,7 +22,15 @@
       "@webiny/app-page-builder/*": ["../app-page-builder/src/*"],
       "@webiny/app-page-builder": ["../app-page-builder/src"],
       "@webiny/lexical-editor/*": ["../lexical-editor/src/*"],
-      "@webiny/lexical-editor": ["../lexical-editor/src"]
+      "@webiny/lexical-editor": ["../lexical-editor/src"],
+      "@webiny/ui": ["../ui/src/*"],
+      "@webiny/ui": ["../ui/src"],
+      "@webiny/feature-flags": ["../feature-flags/src/*"],
+      "@webiny/feature-flags": ["../feature-flags/src"],
+      "@webiny/app-page-builder-elements": ["../app-page-builder-elements/src/*"],
+      "@webiny/app-page-builder-elements": ["../app-page-builder-elements/src"],
+      "@webiny/react-composition": ["../react-composition/src/*"],
+      "@webiny/react-composition": ["../react-composition/src"]
```

Comment:
These paths are wrong. Compare with the ones aboe, which are correct. Even better, run the script to generate these for you:

```
node scripts/generateTsConfigsInPackages.js
```